scripts/irods/controller.py

```diff
@@ -213,11 +215,16 @@ def get_binary_to_pids_dict(self, binaries=None):
                 self.config.server_executable,
                 self.config.rule_engine_executable,
                 self.config.xmsg_server_executable]
+
         d = {}
-        for b in binaries:
-            pids = lib.get_pids_executing_binary_file(b)
-            if pids:
-                d[b] = pids
+        root_pid = lib.get_server_root_pid()
```

Comment:
this may need to call the `with_retry` version to future proof the code. not sure yet.